tests/test_sft_trainer.py

```diff
@@ -68,9 +68,9 @@
 BASE_LORA_KWARGS["peft_method"] = "lora"
 
 BASE_FT_KWARGS = copy.deepcopy(BASE_PEFT_KWARGS)
-BASE_FT_KWARGS["peft_method"] = ""
-BASE_FT_KWARGS["prompt_tuning_init"] = ""
-BASE_FT_KWARGS["prompt_tuning_init_text"] = ""
+BASE_FT_KWARGS["peft_method"] = None
+del BASE_FT_KWARGS["prompt_tuning_init"]
```

Comment:
Just FYI - we dont expect users to pass these keys with full fine tuning . But even if they are set, as long as peft_method is None, doesnt make a difference. Since this section is about setting correct args for testing , we should set what is expected